docs/python/index.rst

```diff
@@ -0,0 +1,167 @@
+.. _sec-python:
+
+========
+ Python
+========
+
+**xNVMe** language bindings for Python provide access the **xNVMe** C API
+access via :ref:`sec-python-ctypes` and :ref:`sec-python-Cython`. Two language
+binding interfaces are provided as they serve to different use-cases.
+
+The :ref:`sec-python-ctypes` are intended for interactive usage when
+experimenting with the **xNVMe** API, that is, for peeking and poking at the
+API and doing non-performance centric administrative tasks with minimal
+dependencies.
+
+The :ref:`sec-python-cython` bindings are intended for Cython applications,
+that is, having **xNVMe** available in your performance-centric Python
+application.
+
+The library bindings are a work-in-progress, provided as a preview of what can
+be done when integrating with Python. See, section :ref:`sec-python-todo`` for
+a list of known short-comings and areas to improve. Your input on these are
+most welcome along with pull-requests to improve the interfaces.
+
+Install
+=======
+
+Before installing, then ensure that the following runtime dependencies are met:
+
+* Python 3.6+
```

Comment:
Yeah, but that is because it only builds on Alpine which has some Python 3.7+ version, however, there are several instances of testing it on Python 3.6 (Leap / Ubuntu), also the ``setup.py`` has ``python_requires=">=3.6"``.